python/apps/taiga/src/taiga/workflows/events/__init__.py

```diff
@@ -28,3 +33,17 @@ async def emit_event_when_workflow_status_is_updated(project: Project, workflow_
         type=UPDATE_WORKFLOW_STATUS,
         content=UpdateWorkflowStatusContent(workflow_status=workflow_status),
     )
+
+
+async def emit_event_when_workflow_status_is_deleted(
+    project: Project, workflow_status: WorkflowStatus, move_to_status_slug: str | None
+) -> None:
+    await events_manager.publish_on_project_channel(
+        project=project,
+        type=DELETE_WORKFLOW_STATUS,
+        content=DeleteWorkflowStatusContent(
+            status_slug=workflow_status.slug,
```

Comment:
could be status and workflow may be a regular "WorkflowStatusSerializer"?

Reply:
That's another option, to recover the workflow from the `WorkflowStatusSerializer`. I'll speak with frontend as it's already coded, but my intention is to return a serializer as it's more standar.
I'll let you know

Reply:
Frontend agreed with this change. Thank you, it's better this way!

```
{
    "type": "event",
    "channel": "projects.8Tw8SByWEe6RYE3yLj9DTQ",
    "event": {
        "type": "workflowstatuses.delete",
        "content": {
            "workflowStatus": {
                "id": "8Tw8TxyWEe6RYE3yLj9DTQ",
                "name": "Done",
                "slug": "done",
                "color": 4,
                "order": 50,
                "workflow": {
                    "name": "Main",
                    "slug": "main"
                }
            },
            "moveToSlug": "ready"
        },
        "correlationId": "9e70741a-de05-41d2-9b73-84f5b227ea5a"
    }
}
```